src/components/home-page/HomePage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import PostList from "../post-list/PostList";
+import {posts} from "../../data/posts";
+
+const HomePage: React.FC = () => {
+    return (
+        <div className="home-page">
+            <PostList posts={posts}/>
+        </div>
+    )
+};
+
+export default HomePage;
```

Comment:
why do you use everywhere "export default"? what is the reason?

Reply:
Because I export only 1 object, so I will be able to simply import it